release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -409,6 +409,29 @@ Be aware that CIDR ranges for networks are not adjustable after cluster installa
 
 If you are using the OVN-Kubernetes cluster network provider, you can now enable IPsec encryption after cluster installation. For more information about how to enable IPsec, see xref:../networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc#configuring-ipsec-ovn[Configuring IPsec encryption].
 
+[id="ocp-4-11-new-metallb-CRDs-logging-control"]
+==== Support for new MetalLB custom resource defintions (CRDs) and control over logging verbosity
+
+MetalLB CRDs have been updated with some new CRDs to support more complex configuration. 
+
+The following new CRDs have been added: 
```

Comment:
You could remove "new" from this sentence for the reason outlined in my preceding comment.